content/docs/integrations/kubernetes/index.md

```diff
@@ -10,423 +10,177 @@ integrationType: OSS plugin
+ℹ️ NOTE: **The API url of the cluster must be available to the Roadie infrastructure services if you are not using the Broker.** You can choose to limit the availablity to our [IP networks](/docs/details/allowlisting-roadie-traffic/). See instructions on setup of the broker in the instructions for your deployment type.
 
-```text
-https://<tenant-name>.roadie.so/administration/settings/plugins/kubernetes
-```
-
-This page describes how to create and set up the API token.
-
-## Steps
+# AWS EKS
 
-In this section we will create an AWS role that will grant Roadie read-only access to your Kubernetes cluster’s resources.
+### Prerequisites
 
-We will use the approach which is recommended by AWS for providing this type of access. To learn more about the AWS concepts used below, you can read the following AWS documentation pages:
+In order to use the Kubernetes plugin for AWS, Roadie requires read-only access via a cross account AWS role in your account, or a Broker connection.
 
+ℹ️ Note: The cross account AWS role is the approach recommended by AWS for this type of access. See the following docs for more context:
  * Assuming Role [learn more](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html)
  * Cross Account Federation [learn more](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_common-scenarios_third-party.html)
  * Kubernetes RBAC [learn more](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)
  * Trust relationship [learn more](https://aws.amazon.com/en/blogs/security/how-to-use-trust-policies-with-iam-roles/)
 
-### Step 1: Creating the cross account federation role
-
-1. Sign into your AWS console and navigate to the [IAM service](https://console.aws.amazon.com/iam/home#/home).
 
-2. Click on ”Role” link (this should be on the left handside of your screen).
+### Setup Guide:
 
-3. Click on the ”Create Role” button.
+You can use a combination of a Cloudformation Template, a Helm Chart and the `eksctl` cli to set this up:
 
-4. Click on ”Another AWS Account” and add the account number on on the Kubernetes configuration page (in Roadie) and then click on ”Next: permissions”.
+ℹ️ NB: Step 1 is only needed per AWS account you add. Steps 2 and 3 will need to be applied to each new cluster you add.
 
-![Another AWS Account](./role-creation.png)
+### Step 1. Create a Cross Account Role (Skip if using the Broker)
+1. Create a new Role in your AWS account's IAM service using this [Cloudformation template](https://console.aws.amazon.com/cloudformation/home#/stacks/create/review?templateURL=https://roadiehq-public-cloudformation-templates.s3.eu-west-1.amazonaws.com/create-role.template&stackName=roadie-role&param_roadieBackendRoleArn=arn:aws:iam::131774410247:role/roadie-backstage-backend-role-rV3ynEHv) NB: You can see the contents of the template by [downloading it here](https://roadiehq-public-cloudformation-templates.s3.eu-west-1.amazonaws.com/create-role.template).
```

Comment:
This url is wrong. The correct url can only be generated at run time. You would need to tell the user to go to the settings page to get the correct link.